packages/react-core/src/helpers/Popper/Popper.tsx

```diff
@@ -25,13 +28,55 @@ export interface PopperProps {
   /** popper direction */
   direction?: 'up' | 'down';
   /** popper position */
-  position?: 'right' | 'left';
+  position?: 'right' | 'left' | 'center';
+  /** Instead of direction and position can set the placement of the popper */
+  placement?: Placement;
   /** The container to append the popper to. Defaults to 'document.body' */
   appendTo?: HTMLElement | (() => HTMLElement);
   /** z-index of the popper element */
   zIndex?: number;
   /** True to make the popper visible */
   isVisible?: boolean;
+  /**
+   * Map class names to positions, for example:
+   * {
+   *   top: styles.modifiers.top,
+   *   bottom: styles.modifiers.bottom,
+   *   left: styles.modifiers.left,
+   *   right: styles.modifiers.right
+   * }
+   */
+  positionModifiers?: {
+    top?: string;
+    right?: string;
+    bottom?: string;
+    left?: string;
+  };
+  /** Distance of the popper to the trigger */
+  distance?: number;
+  /** Callback function when mouse enters trigger */
+  onMouseEnter?: (event?: MouseEvent) => void;
+  /** Callback function when mouse leaves trigger */
+  onMouseLeave?: (event?: MouseEvent) => void;
+  /** Callback function when trigger is focused */
+  onFocus?: (event?: FocusEvent) => void;
+  /** Callback function when trigger is blurred (focus leaves) */
+  onBlur?: (event?: FocusEvent) => void;
+  /** Callback function when trigger is clicked */
+  onTriggerClick?: (event?: MouseEvent) => void;
+  /** Callback function when Enter key is used on trigger */
+  onTriggerEnter?: (event?: KeyboardEvent) => void;
+  /** Callback function when popper is clicked */
+  onPopperClick?: (event?: MouseEvent) => void;
+  /** Callback function when document is clicked */
+  onDocumentClick?: (event?: MouseEvent, triggerElement?: HTMLElement) => void;
+  /** Callback function when keydown event occurs on document */
+  onDocumentKeyDown?: (event?: KeyboardEvent) => void;
+  /** Enable to flip the popper when it reaches the boundary */
+  enableFlip?: boolean;
+  /** The behavior of how the popper flips when it reaches the boundary */
+  flipBehavior?: 'flip' | ('top' | 'bottom' | 'left' | 'right')[];
+  // boundary?: 'scrollParent' | 'window' | 'viewport' | HTMLElement;
```

Comment:
did you mean to leave this here? or uncomment it?

Reply:
i'll remove it, thanks